core/server/data/migrations/versions/5.0/2022-03-14-12-33-delete-duplicate-offer-redemptions.js

```diff
@@ -0,0 +1,22 @@
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const result = await knex.raw(`
+            DELETE
+                duplicate_redemptions
+            FROM
+                offer_redemptions duplicate_redemptions,
```

Comment:
I think we need `as` here

Reply:
It's optional, but I'll add it in for clarity!